### DISCUSSION TITLE
Data Source Name on Meetings Page

### DISCUSSION THREAD

**Discussion by jimmedlen** [code4recovery/12-step-meeting-list#1578] [Q&A]
Our Area Website currently has Outside Sources that are imported.
I would like to display the Data Source Name as a column on the WP Meeting and Edit Meeting pages.

https://area22indiana.org/wp-admin/edit.php?post_type=tsml_meeting

https://area22indiana.org/wp-admin/post.php?post=91217&action=edit

The Alert does not identify the Source.
This meeting was imported from an external data source. Any changes you make here will be overwritten when you refresh the data.

![image](https://github.com/user-attachments/assets/7f82f124-a674-40b4-9062-db703cf4c76e)

![image](https://github.com/user-attachments/assets/03c508cf-38fe-4ce9-845c-364d14be7adf)

<img width="435" alt="image" src="https://github.com/user-attachments/assets/a9272218-3322-4df8-9006-cf150455c028">

**Comment by anchovie91471** [comment]
We don't have this feature at present. I will discuss this with the team and see what we might do. 

There are ways to customize your them to add arbitrary columns to the meeting list screen but it requires a familiarity with php, wordpress and of course TSML.

The edit screen is a bit more tricky and will probably require changes to the plugin files.

**Reply by joshreisner** [reply]
uh well we just added a feature for this actually, unless i'm misreading. if the remote sites use the "entity information" panel on their meetings > settings screen, then the info will show up on the meeting details page on your website

**Comment by anchovie91471** [comment]
@jimmedlen We have considered just removing meetings from the meetings admin page since they can't be edited (they can, but any edits will just be overwritten when the import runs again). Can you give us a little more context about how you'd use the information?

Would it be more helpful to remove the meetings that are imported (since you aren't supposed to edit them anyway) or would you rather see something like a "read only" list where imported meetings are there, but there's no edit link? Obviously, if we did this, we could list the data source in a column.

**Comment by jimmedlen** [comment]
Hello,There are two district web sites within our area set as imported sources. Previously I had all of the meetings entered on the area web site which later caused duplication and a race condition where the area site linked a meeting to the meeting guide app and did not allow the district site to make changes to the meeting information in the App. After adding the most recent imported source I had to open each and every meeting from multiple regions to find which version of a meeting was imported and which was the old duplicate, based on the alert message that the meeting was from an imported source but not which source it was from. I would like to be able to quickly identify meetings from the imported sources to avoid duplication and to direct meeting update requests to the appropriate source. The overlap report from the Meeting Guide Dashboard is fairly large. Thanks,Jim Medlen Sent from my iPhoneOn Nov 29, 2024, at 7:27 PM, Anthony Baggett ***@***.***> wrote:﻿
@jimmedlen We have considered just removing meetings from the meetings admin page since they can't be edited (they can, but any edits will just be overwritten when the import runs again). Can you give us a little more context about how you'd use the information?
Would it be more helpful to remove the meetings that are imported (since you aren't supposed to edit them anyway) or would you rather see something like a "read only" list where imported meetings are there, but there's no edit link? Obviously, if we did this, we could list the data source in a column.

**Reply by joshreisner** [reply]
hey @jimmedlen thanks for describing the use-case. question for you: do you need to see both the local and the imported version of the meeting together in the same list to know which ones are duplicates, or would you know in some other way, such as by region?

reason i'm asking: we're asking ourselves why we display imported meetings at all in this list. if we were not to display these, then all the meetings you'd see in the list would be meetings entered locally. 

would this help your situation, or do you need to see both versions in the list to be confident that there's a copy of the meeting that's being imported?

**Comment by jimmedlen** [comment]
It definitely helps to see the imported meetings. I used the region to find meetings in the overlapping areas.Then I would open each meeting to see if the alert message was at the top to determine which meeting was a locally entered  duplicate meeting. The issue I was hoping to resolve was the need to drill into each  meeting to determine if the meeting was from an imported source. I discovered that the exported csv file has the Data Source Name and URL.I could open a spread sheet to determine the duplicates if needed.Some one in this thread mentioned The Entity data should be displayed but I am not sure where to look for it.Thanks,Jim ***@***.*** Dec 2, 2024, at 1:18 PM, Josh Reisner ***@***.***> wrote:﻿
hey @jimmedlen thanks for describing the use-case. question for you: do you need to see both the local and the imported version of the meeting to know which ones are duplicates, or would you know in some other way, such as by region?
reason i'm asking: we're asking ourselves why we display imported meetings at all in this list. if we were not to display these, then all the meetings you'd see in the list would be meetings entered locally.
would this help your situation, or do you need to see both versions in the list to be confident that there's a copy of the meeting that's being imported?
